src/main/java/ligma/Storage.java

```diff
@@ -0,0 +1,105 @@
+package ligma;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import ligma.task.Task;
+
+/**
+ * This class represents the storage for a particular
+ * Ligma program. The file f is where all data is written to
+ * and read from during the execution of the Ligma program.
+ */
+public class Storage {
```

Comment:
I feel that this class is relatively easy to read and understand. I like how each method are relatively short and easy to interpret since the naming basically helps to explain the main purpose of the method. 👍